menu/ripe79.json

```
@@ -0,0 +1,20 @@
+{
+	"version": 2019100802,
+	"url": "https://ripe79.ripe.net/wp-admin/admin-post.php?action=mps_get_agenda_ics",
+	"title": "RIPE 79",
```

```suggestion
	"id": "RIPE 79 Agenda",
	"title": "RIPE 79",
```

Still needed because of https://github.com/Wilm0r/giggity/issues/134 . My crude CI check would've reminded you of it but it checks for file extension ".ics".